extensions/wikia/AdEngine/js/ml/model/decisionTreeClassifier.js

```diff
@@ -0,0 +1,70 @@
+/*global define*/
+define('ext.wikia.adEngine.ml.model.decisionTreeClassifier', [
+	'wikia.log'
+], function () {
+	'use strict';
+
+	function buildUrl(modelId) {
+		return '/wikia.php?controller=AdEngine2Api&method=getModelData&id=' + modelId
+	}
+
+	function findMax(nums) {
```

Comment:
How does it differ from [`Math.max()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/max)?

Reply:
It returns an index

Reply:
:man_facepalming: you're right ;-)